runelite-api/src/main/java/net/runelite/api/VarPlayer.java

```diff
@@ -56,6 +56,8 @@
 	SPECIAL_ATTACK_PERCENT(300),
 	SPECIAL_ATTACK_ENABLED(301),
 
+	AUTO_RETALIATE(172),
```

Comment:
AUTO_RETALIATE is way too broad, and could be very easily misunderstood, it should describe what its doing, and if complex enough, it should have a java doc.